### PR TITLE
Added GetProcessorFromConfigs variants for displays and views.

### PR DESCRIPTION
**Pull request by num3ric**
Yield a single processor from a color space in config (a) to a display-view in config (b), through the interchange space.

Let me know if we think we need test coverage for these.

### CODE REVIEW

File: src/OpenColorIO/Config.cpp [hunk 1]
```diff
@@ -4957,14 +4957,148 @@ ConstProcessorRcPtr Config::GetProcessorFromConfigs(const ConstContextRcPtr & sr
     }
 
     auto p2 = dstConfig->getProcessor(dstContext, dstExCs, dstColorSpace);
+    if (!p2)
```
Comment (num3ric) [inline src/OpenColorIO/Config.cpp:4960]: Was incorrectly p1 here, so I fixed it causing the strange diff.

**Comment by num3ric** [comment]
(Would be good for the library to have a `.clang-format` file.)

**Comment by remia** [comment]
Thanks @num3ric, sounds like a good idea to add some unit tests inspired from the existing tests on other `GetProcessorFromConfigs()`. I think we should create an issue for the clang-format is not already existing (didn't found one), it was discussed some time ago but got lost.

**Comment by remia** [comment]
There is a conflict on the branch following to recent merges in main.

**Comment by num3ric** [comment]
Understood, back from summer break - will revisit either this week or the next.

**Comment by num3ric** [comment]
Added unit tests. Also moved the direction argument last since in this case it reverses the entire chain of transforms.

**Review by doug-walker — APPROVED** [review]
Thanks Eric!  No need to make any changes (the two notes are just to myself, for later).

**Review by michdolan — APPROVED** [review]

**Pull request merged**